### DISCUSSION TITLE
jumphost/bastation support for rdp

### DISCUSSION THREAD

**Discussion by Ondjultomte** [hoophq/hoop#501] [Ideas]
Many jumphosts are used with a browser to access internal mgmt appliances webb interfaces. A rdp session is used to connect to the jumphost, support for rdp would be nice.

**Comment by buzzzo** [comment]
It might not be the hoop's goal (at least not soon) to be a fully fledged browser based solution.
What could be interesting is to publish an rdp tcp port forwarding connection and generate a .rdp file on the fly fron the hoop connect as soon as the connection is established, then run the default rdp viewer  (on the local machine).

This is for example what rport does and would be very easy to accomplish.
